cpp/build-support/run-test.sh

```diff
@@ -124,6 +119,42 @@ function run_test() {
   fi
 }
 
+function print_coredumps() {
+  # The script expects core files relative to the build directory with unique
+  # names per test executable because of the parallel running. So the corefile
+  # patterns must be set with prefix `core.{test-executable}*`:
+  #
+  # In case of macOS:
+  #   sudo sysctl -w kern.corefile=core.%N.%P or simply core
+  # On Linux:
+  #   sudo echo "core.%e.%p" > /proc/sys/kernel/core_pattern
+  #
+  # and the ulimit must be increased:
+  #   sudo ulimit -c unlimited
+
+  # filename is truncated to the first 15 characters in case of linux, so limit
+  # the pattern for the first 15 characters
+  FILENAME=$(basename "${TEST_EXECUTABLE}")
+  FILENAME=$(echo ${FILENAME} | cut -c-15)
+  PATTERN="^core\.${FILENAME}"
+
+  COREFILES=$(ls | grep $PATTERN)
+  if [ -n "$COREFILES" ]; then
+    echo "Found core dump. Printing backtrace and moving the coredump under ./cores"
```

Comment:
Is "moving the coredump under ./cores" a follow-up task?

Reply:
It was from a previous iteration, now I'm removing the coredump. Will remove the comment.